sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveSQLViewSuite.scala

```diff
@@ -181,4 +181,43 @@ class HiveSQLViewSuite extends SQLViewSuite with TestHiveSingleton {
       }
     }
   }
+
+  test("hive partitioned view is not supported") {
```

Comment:
@cloud-fan I didn't find the right location for this test case. So I put it here for now.